src/layouts/Layout.astro

```diff
@@ -39,7 +33,6 @@ const { title } = Astro.props;
 (function() {
   const prefersDark = window.matchMedia && window.matchMedia('(prefers-color-scheme: dark)').matches
   const setting = localStorage.getItem('theme') || 'auto'
-  if (setting === 'dark' || (prefersDark && setting !== 'light'))
-    document.documentElement.classList.toggle('dark', true)
+  if (setting === 'dark' || (prefersDark && setting !== 'light')) document.documentElement.classList.toggle('dark', true)
```

Comment:
This formatting code may need to be removed.

Reply:
Copy that. Looks like IDE reformatted the code without my notice. Would be more careful in the future.

Reply:
> Copy that. Looks like IDE reformatted the code without my notice. Would be more careful in the future.

Fine， Thanks a lot for your work ❤️